jobs/aws/eks-anywhere-build-tooling/aws-image-builder-presubmits.yaml

```diff
@@ -44,14 +50,11 @@ presubmits:
           if $(make check-project-path-exists); then make build -C $PROJECT_PATH; fi
         env:
         - name: PROJECT_PATH
-          value: projects/aws/image-builder
+          value: "projects/aws/image-builder"
```

Comment:
I assume this won't make a difference, but is there a reason to make the string explicit (i.e with "")?

Reply:
@markapruett all the project paths in template set up in a way that these values come out as strings